ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -546,64 +600,14 @@ private void readFromFile(File fileToRead, Journal journal) throws StoreExceptio
     CrcInputStream crcStream = new CrcInputStream(new FileInputStream(fileToRead));
     DataInputStream stream = new DataInputStream(crcStream);
     try {
-      short version = stream.readShort();
+      version = stream.readShort();
       switch (version) {
```

Comment:
just that I need to capture unsupported versions. I have modified to if else block.